.github/CODEOWNERS

```diff
@@ -80,7 +80,7 @@
 
 # RLlib.
 /rllib/ @sven1977 @avnishn @arturniederfahrenhorst @smorad @maxpumperla @kouroshhakha
-/doc/source/rllib/ @sven1977 @avnishn @arturniederfahrenhorst @smorad @maxpumperla @kouroshhakha @ray-project/ray-docs
+/doc/source/rllib/ @sven1977 @avnishn @arturniederfahrenhorst @maxpumperla @kouroshhakha @simonsays1980 @ray-project/ray-docs
```

Comment:
seems that someone needs to add @simonsays1980 into a group first?

@sven1977 , sorry, but I do not really have context on who this is.

Reply:
All good, @zhe-thoughts already approved :)
Thanks anyways @aslonnie !